api/src/main/java/javax/enterprise/context/spi/Context.java

```diff
@@ -58,7 +58,7 @@
      * 
      * @return the scope
      */
-    public Class<? extends Annotation> getScope();
```

Comment:
All methods on interfaces are public by default. What is the point to change them?